iset.mm

```diff
@@ -95522,18 +95595,32 @@ the previous value and the value of the input sequence (second operand).
     $d .+ w x y z $.  $d F w x y z $.  $d M w x y z $.  $d N x y $.
     $d S w x y z $.  $d ph w x y z $.
     iseqcl.1 $e |- ( ph -> N e. ( ZZ>= ` M ) ) $.
-    iseqcl.ex $e |- ( ph -> S e. V ) $.
     iseqcl.2 $e |- ( ( ph /\ x e. ( ZZ>= ` M ) ) -> ( F ` x ) e. S ) $.
     iseqcl.3 $e |- ( ( ph /\ ( x e. S /\ y e. S ) ) -> ( x .+ y ) e. S ) $.
     $( Closure properties of the recursive sequence builder.  (Contributed by
        Jim Kingdon, 1-Jun-2020.) $)
     iseqcl $p |- ( ph -> ( seq M ( .+ , F , S ) ` N ) e. S ) $=
-      ( vz vw cfv wcel cv c1 co cuz cseq caddc cmpt2 cop cfrec cmpt eluzel2 syl
-      cz eqid wral uzid ralrimiva wceq fveq2 eleq1d rspcv sylc iseqovex iseqval
-      frecuzrdgcl mpdan ) AHGUAPZQZHDEFGUBZPEQJABCGFPZHGBCVDEBRZSUCTZVHCRNOVDEO
-      RNRSUCTFPDTUDZTUEUDGVGUEUFZEVFVJBUJVIUGGUFZIAVEGUJQZJGHUHUIZVLUKKAGVDQZVH
-      FPZEQZBVDULVGEQZAVMVOVNGUMUIAVQBVDLUNVQVRBGVDVHGUOVPVGEVHGFUPUQURUSABCNOD
-      EFGLMUTVKUKZABCNODVKEFGVSLMVAVBVC $.
+      ( vz vw cfv cv c1 caddc co cmpt2 wcel cuz cseq cop cfrec cmpt eluzel2 syl
+      cz eqid wceq fveq2 eleq1d ralrimiva rspcdva iseqovex iseqval frecuzrdgtcl
+      uzid ffvelrnd ) AGUANZEHDEFGUBZABCGFNZGBCUTEBOZPQRZVCCOLMUTEMOLOPQRFNDRSZ
+      RUCSGVBUCUDZEVAVEBUHVDUEGUDZAHUTTGUHTZIGHUFUGZVGUIAVCFNZETZVBETBUTGVCGUJV
+      JVBEVCGFUKULAVKBUTJUMAVHGUTTVIGURUGUNABCLMDEFGJKUOVFUIZABCLMDVFEFGVLJKUPU
+      QIUS $.
+  $}
+
+  ${
+    $d .+ w x y z $.  $d F w x y z $.  $d M w x y z $.  $d N x y $.
+    $d S w x y z $.  $d ph w x y z $.
+    iseqclOLD.1 $e |- ( ph -> N e. ( ZZ>= ` M ) ) $.
+    iseqclOLD.ex $e |- ( ph -> S e. V ) $.
+    iseqclOLD.2 $e |- ( ( ph /\ x e. ( ZZ>= ` M ) ) -> ( F ` x ) e. S ) $.
+    iseqclOLD.3 $e |- ( ( ph /\ ( x e. S /\ y e. S ) ) -> ( x .+ y ) e. S ) $.
+    $( Closure properties of the recursive sequence builder.  New proofs should
+       use ~ iseqcl instead (which is the same with one fewer hypotheses).
+       (Contributed by Jim Kingdon, 1-Jun-2020.)  (New usage is discouraged.)
```

Comment:
property (?)
hypothesis

Reply:
I fixed this (two places) and made a few other small comment edits. Will merge as soon as the tests pass.